tests/transaction/test_get_list_transactions.py

```diff
@@ -0,0 +1,368 @@
+"""
+Provide tests for command line interface's get list of transactions command.
+"""
+import json
+
+import pytest
+from click.testing import CliRunner
+
+from cli.constants import (
+    FAILED_EXIT_FROM_COMMAND_CODE,
+    NODE_IP_ADDRESS_FOR_TESTING,
+    PASSED_EXIT_FROM_COMMAND_CODE,
+)
+from cli.entrypoint import cli
+from cli.utils import dict_to_pretty_json
+
+
+def test_get_list_transactions_with_ids():
+    """
+    Case: get a list transactions by ids.
```

Comment:
`ids` > `identifiers`. Check other tests for this.

Reply:
@dmytrostriletskyi, done.